mmv1/templates/terraform/examples/integration_connectors_endpoint_attachment.tf.erb

```diff
@@ -0,0 +1,12 @@
+data "google_project" "test_project" {
+}
+
```

Comment:
nit: could this be removed? It's not used elsewhere in the example and will cause the acceptance test that is generated from this example to make an unnecessary API call